### PR TITLE
Created button component

### PR DESCRIPTION
**Pull request by pritam-kr**
Created button component of multiple types

- Primary Button
- Outline Button
- Button with Icon
- Floating action button
- Text button

and did some more change in responsive in mobile.

### CODE REVIEW

File: components/Button.css [hunk 1]
```diff
@@ -0,0 +1,135 @@
+.component-button {
+    margin-top: 2em;
+}
+
+.button-wrapper {}
```
Comment (srishti-maurya) [inline components/Button.css:5]: maybe you can remove this part, if not in use

File: components/Button.css [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    background: var(--btn-warning-hover);
+}
+
+.btn-bark {
+    background: var(--btn-dark);
+    color: var(--text-color);
+}
+
+.btn-dark:hover {
+    background: var(--btn-dark-hover) !important;
```
Comment (srishti-maurya) [inline components/Button.css:85]: i guess it's suggested to not use !important, maybe you find an alternative way to apply the style

File: components/Button.css (hunk 1 above)
Comment (pritam-kr) [reply components/Button.css:5]: I removed this.

File: components/Button.css (hunk 2 above)
Comment (pritam-kr) [reply components/Button.css:85]: Sorry I forgot to remove this

File: docscss/docs.css [hunk 3]
```diff
@@ -67,31 +74,60 @@
...
     display: flex;
-    justify-content: flex-end;
+    justify-content: space-between;
     align-items: center;
-    box-shadow: 0 2px 2px -2px rgba(0, 0, 0, .2);
+    box-shadow: var(--box-shadow);
+    /* position: sticky;
+    right: 0;
+    left: 0;
+    top: 0; */
```
Comment (srishti-maurya) [inline docscss/docs.css:86]: it'll be better if you remove the commented styles to have a clean looking stylesheet

**Review by srishti-maurya — APPROVED** [review]

File: docscss/docs.css (hunk 3 above)
Comment (pritam-kr) [reply docscss/docs.css:86]: I commented some lines for more work.

Comment (srishti-maurya) [reply docscss/docs.css:86]: oh in that case it's fine then! btw great work👍

**Review by manojasarsa — APPROVED** [review]
Hey Pritam, looks awesome! Just a small suggestion, if you can give little margins to buttons ( mainly for Floating action button). They will look great!

**Comment by preethasri** [comment]
Hey pritam looks nice

**Review by techieshubham30 — APPROVED** [review]

**Pull request merged**